### PR TITLE
Modernize the SHIELD Genesis Kit

### PR DESCRIPTION
**Pull request by jhunt**
This PR modernizes the SHIELD Genesis Kit so that it handles things a
bit more ergonomically, using new features available as of Genesis 2.6.0
and above.

This has been vetted manually in both vSphere and AWS environments.

### CODE REVIEW

File: MANUAL.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+- `shield_disk_pool` - The persistent disk pool that Vault VMs will
+  use.  This pool must exist in your cloud config.  Defaults to
+  `shield`.
+
+- `shield_vm_type` - What type of VM to deploy.  This type must
+  exist in your cloud config.  Defaults to `small`.
+
+- `shield_network` - What network to deploy SHIELD into.  This
+  network must be defined in your cloud config.  Defaults to
+  `small`.
```
Comment (Proplex) [inline MANUAL.md]: Small typo, it should be `shield` not `small`

**Comment by Proplex** [comment]
With the exception of the minor documentation typo, everything looks great. Tested on both GCP and vSphere.

**Comment by jhunt** [comment]
@Proplex feedback addressed!

File: MANUAL.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+# Base Parameters
+
+- `shield_static_ip` - The static IP address of the SHIELD core.
+  This must be designated as static in the chosen network's cloud
+  config definition.
+
+- `installation` - The name of the SHIELD installation, as
+  reported from inside the SHIELD UI.
+  Defaults to `S.H.I.E.L.D. Alpha`
```
Comment (dennisjbell) [inline MANUAL.md]: Actually this defaults to `S.H.I.E.L.D.`

Comment (dennisjbell) [reply MANUAL.md]: Actually, it technically doesn't default to anything (ie it is not defined), but if not present, things that look it up default to it just being `S.H.I.E.L.D.`

File: MANUAL.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+  `shield`.
+
+- `stemcell_os` - The operating system you want to deploy SHIELD
+  on.  This defaults to `ubuntu-trusty`.
+
+- `stemcell_version` - The version of the stemcell to deploy.
+  Defaults to `latest`, which is usually what you want.
+
+- `availability_zones` - What BOSH HA availability zones to deploy
+  the Vault across.  The chosen network must have at least one
```
Comment (dennisjbell) [inline MANUAL.md]: C&P error: Vault

File: hooks/info [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+describe "#B{$(lookup params.installation 'S.H.I.E.L.D.')}"
+    echo
+describe "endpoint information"
+describe "  #C{https://$(lookup params.shield_static_ip)}"
```
Comment (dennisjbell) [inline hooks/info]: Recommendation: use exodus url instead of https:// + params.shield_static_ip

File: MANUAL.md (hunk 2 above)
Comment (jhunt) [reply MANUAL.md]: :+1:

File: MANUAL.md (hunk 3 above)
Comment (jhunt) [reply MANUAL.md]: :+1:

File: hooks/info (hunk 4 above)
Comment (jhunt) [reply hooks/info]: :+1:

**Pull request merged**